src/codemagic/tools/_app_store_connect/action_groups/beta_groups_action_group.py

```diff
@@ -0,0 +1,82 @@
+from typing import Sequence
+from typing import Union
+
+from codemagic import cli
+from codemagic.apple import AppStoreConnectApiError
+from codemagic.apple.app_store_connect.builds import Builds
+from codemagic.apple.app_store_connect.provisioning.beta_groups import BetaGroups
+from codemagic.apple.resources import Build
+from codemagic.apple.resources import ResourceId
+from codemagic.cli import Colors
+
+from ..abstract_base_action import AbstractBaseAction
+from ..action_group import AppStoreConnectActionGroup
+from ..arguments import BuildArgument
+from ..errors import AppStoreConnectError
+
+
+class BetaGroupsActionGroup(AbstractBaseAction):
+
+    @cli.action('add-build',
+                BuildArgument.BUILD_ID_RESOURCE_ID_REQUIRED,
+                BuildArgument.BETA_GROUP_NAMES_REQUIRED,
+                action_group=AppStoreConnectActionGroup.BETA_GROUPS)
+    def add_build(self, build_id: Union[ResourceId, Build], beta_group_names: Sequence[str]):
+        """
+        Add build to a Beta group
+        """
+        builds_manager = Builds(client=self.api_client)
+        app = builds_manager.read_app(build_id)
```

Comment:
Here and elsewhere use the propertis on [`AppStoreConnectApiClient`](https://github.com/codemagic-ci-cd/cli-tools/blob/master/src/codemagic/apple/app_store_connect/api_client.py#L146):
```suggestion
        app = self.api_client.builds.read_app(build_id)
```